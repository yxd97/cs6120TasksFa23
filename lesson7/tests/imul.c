#include <stdio.h>
#include <stdlib.h>
// ARGS: 2
int main(int argc, char** argv) {
    if (argc < 2) {
        return 1;
    }
    int x = atoi(argv[1]);
    int r1 = x * 3;
    printf("%d x %d = %d\n", x, 3, r1);
    return 0;
}
