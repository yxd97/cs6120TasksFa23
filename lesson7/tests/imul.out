2 x 3 = 6
